src/api.h

```diff
@@ -434,3 +432,58 @@ return __p.invoke(func);\
             return p;                                                                           \
         })()
 }
+
+inline int major(int version)
```

Comment:
There were comments that major/minor are commonly used in libraries/macros resulting in conflicts, #2665.
Even though it is encapsulated in namespace I'd consider more unique "lrs_version_major" or similar